src/WorkflowGui/Controller/WorkflowController.php

```diff
@@ -16,10 +16,14 @@
 
 namespace Youwe\Pimcore\WorkflowGui\Controller;
 
+use DataDictionaryBundle\Graph\Presenters\GraphViz;
```

Comment:
DataDictionaryBundle is not a dependency we have

Reply:
Removed this - I do not need it anyway.